doc/classes/Material.xml

```diff
@@ -5,6 +5,9 @@
 	</brief_description>
 	<description>
 		Material is a base [Resource] used for coloring and shading geometry. All materials inherit from it and almost all [VisualInstance3D] derived nodes carry a Material. A few flags and parameters are shared between all material types and are configured here.
+		[b]Note:[/b]  For performance reasons the [code]changed[/code] signal, it's only emitted when these properties are changed:
```

Comment:
```suggestion
		[b]Note:[/b]  For performance reasons the [signal Resource.changed] signal, it's only emitted when these properties are changed:
```

Reply:
<s>I think my review above is enough. :stuck_out_tongue: </s>

I'm a imbecile who didn't commit his review. :upside_down_face:

Reply:
done!